test/kdump_test.rb

```diff
@@ -729,4 +729,29 @@
       expect(memory).to eq(high: "64", low: "32")
     end
   end
+
+  describe ".AutoYaST" do
+    before do
+      Yast::Mode.SetMode(mode)
+    end
+
+    context "during profile import" do
+      let(:mode) { "autoinstallation" }
+      let(:profile) { {"add_crash_kernel"=>true,
+                        "crash_kernel"=>"256M",
+                        "general"=>{"KDUMP_SAVEDIR"=>"file:///var/dummy"}
+                      }
+                    }
+      # bnc#995750
+      it "imported values will not be overwritten by the proposal" do
+        Yast::Kdump.Import(profile)
+        Yast::Kdump.Propose
+        ret = Yast::Kdump.Export
```

Comment:
NP: is it possible to check the current values of "add_crash_kernel", "crash_kernel", etc. without calling `#Export`?. As said, this is probably too nickpicking but if we are trying to check that the sequence Import && Propose works as expected, there is no need to introduce Export in the equation. Part of my obsession about keeping unit tests minimal and related only to the tested method. :-)